web/gui/index.html

```diff
@@ -1157,7 +1157,9 @@ <h4 class="modal-title" id="updateModalLabel">Update Check</h4>
                         <hr/>
                     </div>
                     <div>
-                        For progress reports and key netdata updates: <strong><a href="https://twitter.com/linuxnetdata" target="_blank">follow netdata on <i class="fab fa-twitter"></i> twitter</a></strong>.
+                        For progress reports and key netdata updates:
+                        <strong><a href="https://community.netdata.cloud/category/1/announcements-and-roadmap">Join the Netdata Community</a></strong> or
+                        <a href="https://twitter.com/linuxnetdata" target="_blank">follow netdata on <i class="fab fa-twitter"></i> twitter</a>.
```

Comment:
@zack-shoylev this file actually should be deleted because it's not used anymore! We get the new dashboard from separate repo, so these changes should actually be added here: https://github.com/netdata/dashboard/blob/master/public/index.html . Please ping me if you'll have any questions